### PR TITLE
Add support for AssFilter DirectShow subtitle provider

### PR DESCRIPTION
**Pull request by Blitzker**
I don't know well the mpc-hc code base, but thoses changes add the "AssFilter" subtitle renderer in the Options -> Output -> Subtitle Renderer.

### CODE REVIEW

File: src/mpc-hc/resource.h [hunk 1]
```diff
@@ -1569,9 +1569,10 @@
 #define IDS_PPAGEADVANCED_SCORE         57530
 #define IDS_PPAGE_FS_CLN_AUDIO_DELAY    57531
 #define IDS_PPAGEADVANCED_DEFAULTTOOLBARSIZE 57532
+#define IDS_SUBTITLE_RENDERER_ASS_FILTER 57533
 
 // Next default values for new objects
-//
```
Comment (kasper93) [inline src/mpc-hc/resource.h]: Please revert this trailing space. It is added by visual studio resource editor. So it needs to stay so it won't pop every time someone edits it.

**Comment by Blitzker** [comment]
Sorry about that. I also added commit 01457be so AssFilter is the only active subtitle renderer when it is selected.

File: src/mpc-hc/resource.h (hunk 1 above)
Comment (Blitzker) [inline src/mpc-hc/resource.h]: Sorry about that. Fixed

**Comment by amayra** [comment]
i like to see AssFilterMod in mpc-hc

**Pull request closed without merging**